README.md

```diff
@@ -83,4 +83,20 @@ if __name__ == "__main__":
   opentracing.tracer.flush()
 ```
 
+### Thrift
+When using apache thrift rpc, make sure to both disable use_http by setting it to False as well
```

Comment:
Can we wrap use_http, use_thrift, False, and True with ticks (`) so that they are more obviously read as code parameters

Reply:
Sure.